WardenApi/src/main/java/com/salesforce/dva/warden/dto/Entity.java

```diff
@@ -0,0 +1,147 @@
+package com.salesforce.dva.warden.dto;
```

Comment:
Header comment needs to start on line 1 in all files.  Anything like this package declaration has to occur after the header.